CommonClasses/interface/PhotonID.h

```diff
@@ -136,7 +136,7 @@ namespace ExoDiPhotons{
       }
     } // end EE
     else {
-      return 99999.99;
+      return -99999.99;
```

Comment:
Is this code ever reached in practice? Since we don't normally care about photons in the EB-EE transition region or those with abs(scEta) > 2.5, the assignment of a large photon isolation offset to these photons should not cause any issues.